CMakeLists.txt

```diff
@@ -9,4 +9,4 @@
 
 cmake_minimum_required( VERSION 3.3.2 FATAL_ERROR )
 
-project( fv3_jedi_data VERSION 1.4.0 DESCRIPTION "FV3-JEDI Test Files" )
+project( fv3_jedi_data VERSION 1.5.0 DESCRIPTION "FV3-JEDI Test Files" )
```

Comment:
[general question, not a change request, not related to this PR] What meaning are we trying to convey with these integers? Do we have criteria for mapping code changes to tiers of the version number? As I understand it, semantic versioning traditionally correlates to API/ABI compatibility breaking changes, but we never even talk about this...

Reply:
You are 100% correct. We should be doing a better job with this in general. In many cases, though, it's still ok what we do. The patch version change obviously refers to bug/hot fixes. The increase in the minor version number indicates no API changes, modest updates. A major version number indicates an overhaul, a significant API change, ...